docs/security_analysis/index.md

```diff
@@ -2,7 +2,7 @@
 
 Psalm can attempt to find connections between user-controlled input (like `$_GET['name']`) and places that we don’t want unescaped user-controlled input to end up (like `echo "<h1>$name</h1>"` by looking at the ways that data flows through your application (via assignments, function/method calls and array/property access).
 
-You can enable this mode with the `--taint-analysis` command line flag. When taint analysis is enabled, no other analysis is performed.
+You can enable this mode with the `--taint-analysis` command line flag. When taint analysis is enabled, no other analysis is performed.  To [ensure accurate results](https://github.com/vimeo/psalm/issues/6156), Psalm should be run normally prior to taint analysis, and any errors should be fixed.
```

Comment:
I'd use the term `comprehensive` instead of `accurate`. Psalm is accurate in what it reports, we don't want users to think they're doing harm, it's just that the list of vulnerability is possibly not complete

Reply:
Hmm... From my perspective in this case, Psalm inaccurately implied that all files were being scanned when they were not, and inaccurately reported "No errors found", which lead to a real risk of harm by approving & deploying code that contained vulnerabilities.

Y'all are in charge of course...  If I don't hear back in a couple of days I'll make the change as requested.

Reply:
The language has been changed as requested.